lib/grape/dsl/inside_route.rb

```diff
@@ -167,6 +167,23 @@ def error!(message, status = nil, additional_headers = nil)
         throw :error, message: message, status: self.status, headers: headers
       end
 
+      # Creates a Rack response based on the provided message, status, and headers.
+      # The content type in the headers is set to the default content type unless provided.
+      # The message is HTML-escaped if the content type is 'text/html'.
+      #
+      # @param message [String] The content of the response.
+      # @param status [Integer] The HTTP status code.
+      # @params headers [Hash] (optional) Headers for the response
+      #                      (default: {Rack::CONTENT_TYPE => content_type}).
+      #
+      # Returns:
+      # A Rack::Response object containing the specified message, status, and headers.
+      #
+      def rack_response(message, status = 200, headers = { Rack::CONTENT_TYPE => content_type })
```

Comment:
As now we are performing the handler under the endpoint instance, the methods called inside the `rescue_from` should be defined in the `inside_route.rb` file. I've experienced problems with the `rack_response` method:

The `rack_response` method is used in several tests inside the `rescue_from`. However, this method is not defined in the `inside_route.rb` file, what causes that a `NoMethodError` is raised.

**Proposed Solution**: Define the `rack_response` method in the `inside_route.rb` file, so now endpoints can also perform this method for building a rack response.